.github/workflows/build_rpm.yaml

```diff
@@ -0,0 +1,174 @@
+          fetch-tags: 'true'
+          ref: 'ci/create-rpm-package'  # TODO Change on merge
+          path: 'packaging'
+
+      - name: Parse alumet version
+        id: params
+        run: |
+          name=$(awk '/Name/ { print $2; exit }' packaging/rpm/specs/alumet.spec)
+          echo "name=$name" >> $GITHUB_OUTPUT
+          echo "Pipeline will build [$name] in version [${{ inputs.build-version }}] for the architecture [${{ inputs.target-architecture }}]" >> $GITHUB_STEP_SUMMARY
+
+  packaging:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    needs:
+      - parsing
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["fedora:40", "fedora:41", "fedora:42", "registry.access.redhat.com/ubi8/ubi:8.3", "registry.access.redhat.com/ubi9/ubi:9.5"]
+    env:
+      name: ${{ needs.parsing.outputs.name }}
+    container:
+      image: ${{ matrix.os }}
+    steps:
+
+      - uses: actions/checkout@v4
+        with:
+          repository: alumet-dev/packaging
+          fetch-tags: 'true'
+          ref: 'ci/create-rpm-package'
+          path: 'packaging'
+
+      - name: Export good OS version
+        run: |
+          echo "OS=$(echo ${{ matrix.os }} | awk -F'[/:]' '{print $(NF-1) "" $NF}')" >> $GITHUB_ENV
+
+      - name: Install RPM on Fedora
+        if: contains(matrix.os, 'fedora')
+        run: |
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs  | sh -s -- -y
+          export PATH=$PATH:$HOME/.cargo/bin/
+          sudo dnf install rpm-build -y
+          sudo dnf install cargo-rpm-macros -y
+          sudo dnf install openssl-devel -y
+
+      - name: Install RPM on other than Fedora
+        if: ${{ ! contains(matrix.os, 'fedora') }}
+        run: |
+          mkdir -p /opt/rustup
+          mkdir -p /opt/cargo
+          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs  | RUSTUP_HOME=/opt/rustup CARGO_HOME=/opt/cargo sh -s -- -y
+          echo 'export PATH=$PATH:/opt/rustup:/opt/cargo/bin' >> ~/.bashrc && source ~/.bashrc
+          echo 'export PATH=$PATH:/opt/rustup:/opt/cargo/bin' >> $GITHUB_PATH
+          packages=("unzip" "rpm-build" "openssl-devel" "curl" "gcc")
+          for package in "${packages[@]}"; do
+            if ! yum list installed "$package" >/dev/null 2>&1; then
+              yum install -y --allowerasing "$package"
+            fi
+          done
+          rustup default stable
+
+      - name: Setup for alumet
+        run: |
+          mkdir -p alumet
+          mkdir -p /etc/alumet  // TODO Change on merge !!
+
+      - name: Checkout alumet repo
+        uses: actions/checkout@v4
+        with:
+          # repository: alumet-dev/alumet
+          repository: AngeCyp/alumet-testing  # TODO Change on merge
+          path: "alumet-agent"
+          ref: '${{ inputs.tag }}'
+
+      - uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: true
+          cache-targets: "alumet"
+
+      - name: get Spec settings
```

Comment:
This look like the RPM build step, maybe reflect it in step title `Build RPM package`